spec/redux-test.js

```diff
@@ -0,0 +1,31 @@
+import * as actions from '../src/redux/actionCreators';
+import * as types from '../src/redux/action-types';
+import running from '../src/redux/reducer';
+
+describe('The action creator', function() {
+  it('run should return return a object w/action.payload === true', function() {
+    var expectedAction = {
+      type: types.RUNNING,
+      payload: true
+    };
+    expect(actions.run()).toEqual(expectedAction);
```

Comment:
so you descriptiong of the test and the actual test are somewhat different. Reading your description, I would have written the test like so:

```javascript
expect(actions.run().payload).toBeTrue();
```

Reading your code, I would have expected the description to be "run should return an action object with the type 'RUNNING' and payload true"

Either way is fine, just try to match the description to the test as best you can.